python/langsmith/client.py

```diff
@@ -4709,15 +4709,10 @@ def update_example(
     def update_examples(
         self,
         *,
-        example_ids: Sequence[ID_TYPE],
-        inputs: Optional[Sequence[Optional[Dict[str, Any]]]] = None,
-        outputs: Optional[Sequence[Optional[Mapping[str, Any]]]] = None,
-        metadata: Optional[Sequence[Optional[Dict]]] = None,
-        splits: Optional[Sequence[Optional[str | List[str]]]] = None,
-        dataset_ids: Optional[Sequence[Optional[ID_TYPE]]] = None,
-        attachments_operations: Optional[
-            Sequence[Optional[ls_schemas.AttachmentsOperations]]
-        ] = None,
+        dataset_id: ID_TYPE | None = None,
+        updates: Optional[List[ls_schemas.ExampleUpdateWithAttachments]] = None,
```

Comment:
we should rename `ExampleUpdateWithAttachments` to `ExampleUpdate` if no name collision with existing classes